mysql/4.initialize-database-db-id.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE new_message (
   payload LONGTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
   published SMALLINT DEFAULT 0,
   creation_time BIGINT
-);
+) AUTO_INCREMENT = <<CURRENT_TIME_IN_MILLISECONDS>>;
```

Comment:
Isn't it this: https://github.com/eventuate-foundation/eventuate-common/blob/a691ea45909ff74f9c19d527402c0859931752ce/eventuate-common-jdbc/src/main/java/io/eventuate/common/jdbc/sqldialect/MySqlDialect.java#L6

Reply:
yes, but it does not work. I am preparing details.